NEWS

```diff
@@ -357,6 +357,9 @@ Open Plans:
 ** in 64-bit environments, the maximum field size was increased from
    268435456 bytes (999999998 bytes for OCCURS UNBOUNDED) to 2 GB
 
+** the call-stack on error / in the dump file now contains all parameters given
+   to the program via command line options, if any
+
```

Comment:
looks to me that either the feature or the testcases for that weren't merged

Reply:
Seems this paragraph was unduly included when merging 4655.

Reply:
Hmm, actually, that feature from GC3 4397 (https://sourceforge.net/p/gnucobol/code/4397) was merged in GC4 at 4401 (https://sourceforge.net/p/gnucobol/code/4401). But the NEWS entry was then removed in GC4 by 4768 (https://sourceforge.net/p/gnucobol/code/4768). Probably a mistake back then ?

Reply:
OK, that explains that at least, so re-merging it in now is the right thing to do. Thanks for checking!